### DISCUSSION TITLE
AgentContextWrapper cannot found application.io.opentelemetry.api.trace.Span

### DISCUSSION THREAD

**Discussion by will-zdu** [open-telemetry/opentelemetry-java-instrumentation#8770] [Q&A]
1.io.opentelemetry.javaagent.instrumentation.opentelemetryapi.context.AgentContextWrapper can not found 
application.io.opentelemetry.api.trace.Span in my ide，I can only found  io.opentelemetry.api.trace.Span in my ide

Maybe is that， The package name for the API that the agent does not invoke should not overlap with the package name used in the client's code. but how did I fix it on ide？

**Comment by mateuszrzeszutek** [comment]
The `application.io.opentelemetry.api...` classes are generated by the [`opentelemetry-api-shaded-for-instrumenting`](https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/main/opentelemetry-api-shaded-for-instrumenting/build.gradle.kts) module. Try running a full `./gradlew assemble` once, these classes should be generated as part of the build process.
